cmd/jujud/agent/engine_test.go

```diff
@@ -133,6 +134,15 @@ var (
 		"unconverted-api-workers",
 		"unit-agent-deployer",
 	}
+
+	alwaysCAASWorkers = []string{
+		"agent",
+		"api-caller",
+		"operator",
```

Comment:
surely "operator" shouldn't be running during migration? or is the TODO because we don't handle migration in the CAAS operator yet?

Reply:
correct, there's no migration checks for CAAS models yet